test/boulder-tools/build.sh

```diff
@@ -39,10 +39,10 @@ wait
 # protoc-gen-go outputs a line that says:
 # const _ = grpc.SupportPackageIsVersion4
 # so it will fail to compile with a different version of the grpc package.
-# Since we currently have version 3 of the grpc package vendored, we have to
+# Since we currently have version 4 of the grpc package vendored, we have to
```

Comment:
This explanation doesn't quite make as much sense now that the grpc package matches the head of protoc-gen-go. Can you rewrite it to reflect the current situation?